examples/mobile/HomeApp/js/app.js

```diff
@@ -339,24 +415,24 @@ import Storage from "./clipping-storage.js";
 		// use apps list from storage or default apps list if sth wrong
 		appsList = storage.readAllFromStorage(Storage.elements.APPSLIST);
```

Comment:
why we need local variable appList if homeApp.appsList is always the same?
if you want just simplify code you should write 
appsList = homeApp.appsList
instead of
homeApp.appsList = appsList;

Reply:
Ok. Done.